website/data/docs-navigation.js

```diff
@@ -199,10 +199,6 @@ export default [
     category: 'builders',
```

Comment:
@nywilken @zchsh  I noticed the docker components are still listed in this file. I removed the amazon ones but not sure if I should have done it. Could some of you confirm which is the correct thing to do regarding this file?

Reply:
Ah good catch @sylviamoss. Is this file still needed @zchsh? I thought with the new navigation structure this file was  obsolete.

Reply:
@nywilken you're correct, it is obselete - it was removed in #10656 but must have been added back in at some point. It can definitely be deleted as it's no longer used - any edits should be made to `website/data/docs-nav-data.json` instead 👍

Reply:
Nice! Thanks, I'll get rid of it 🗑️

Reply:
☝️Also just to confirm, I re-ran the migration from `docs-navigation.js` to `docs-nav-data.json`, and their contents are identical apart from the split-out of `docker` in the latter. So definitely safe to remove `docs-navigation.js` 🔪